Intersect.Client.Core/Interface/Game/Inventory/InventoryItem.cs

```diff
@@ -269,72 +258,55 @@ private void _iconImage_DoubleClicked(Base sender, MouseButtonState arguments)
         }
     }
 
-    private void _iconImage_Clicked(Base sender, MouseButtonState arguments)
+    private void IconImage_Clicked(Base sender, MouseButtonState arguments)
     {
-        switch (arguments.MouseButton)
+        if (arguments.MouseButton is MouseButton.Right)
         {
-            case MouseButton.Left:
-                _clickTime = Timing.Global.MillisecondsUtc + 500;
-                break;
-
-            case MouseButton.Right:
-                if (ClientConfiguration.Instance.EnableContextMenus)
+            if (ClientConfiguration.Instance.EnableContextMenus)
+            {
+                OpenContextMenu();
```

Comment:
add a return after this line inside of the if statement so the `else` can be un-nested